web/cobrands/highways/assets.js

```diff
@@ -70,7 +67,7 @@ fixmystreet.assets.add(defaults, {
                 // already shown the HE message)
                 if ($('#js-highways:checked').length) {
                     if (category && !category.match('NH')) {
-                        non_he_selected();
+                        he_council_litter_cat_selected();
```

Comment:
We need a new function in this case; using non_he_selected meant the right subcategory was picked but the top-level National Highways was unselected, which confused future category changes (e.g. reportedCategory would be blank next category change and the NH would always toggle). This new function does the same as non_he_selected but maintains the top level selection.